### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by gongweibao**
Fix bugs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,21 +70,30 @@ Then open your browser and visit http://cloud.paddlepaddle.org.
...
 ```
+cd paddlecloud
 npm install
 pip install -r requirements.txt
 ./manage.py migrate
 ./manage.py loaddata sites
 npm run dev
 ```
 
+If `npm` haven't been installed, you need to
```
Comment (typhoonzero) [inline README.md]: The command may differ for different OSs, just add one line in Pre-Requirements: "nodejs"

File: README.md [hunk 2]
```diff
@@ -33,10 +33,10 @@
   mkdir -p /home/pcloud/data/mysql
   mkdir -p /home/pcloud/data/certs
   ```
-  - Copy Kubernetes CA files (ca.pem, ca-key.pem, ca.srl) to `pcloud_data/certs` folder
-  - Copy Kubernetes admin user key (admin.pem, admin-key.pem) to `pcloud_data/certs` folder
-  - Copy CephFS Key file(admin.secret) to `pcloud_data/certs` folder
-  - Copy `/paddlecloud/settings.py` file to `pcloud_data` folder
+  - Copy Kubernetes CA files (ca.pem, ca-key.pem, ca.srl) to `pcloud/data/certs` folder
```
Comment (typhoonzero) [inline README.md]: Maybe not put strict folders here, the folders are used by `k8s/cloud_deployment.yaml`, and these folders need to be full path begin from `/`. So users need to define their own hostpath for `yaml` files to use.

File: README.md (hunk 1 above)
Comment (gongweibao) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (gongweibao) [inline README.md]: Done

File: paddlecloud/requirements.txt [hunk 3]
```diff
@@ -9,3 +9,4 @@ kubernetes
 MySQL-python
 djangorestframework
 django_sendmail_backend
+nodejs
```
Comment (typhoonzero) [inline paddlecloud/requirements.txt]: `Dockerfile` already have this.

**Pull request merged**